src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonWriterTests.cs

```diff
@@ -8069,6 +7603,33 @@ public static IEnumerable<object[]> JsonEncodedTextStrings
                 };
             }
         }
+
+        private static IEnumerable<JsonWriterOptions> JsonOptions()
+        {
+            return from indented in new[] { true, false }
+                   from skipValidation in new[] { true, false }
+                   from indentText in indented ? new[] { null, "", " ", "  ", "    ", "\t" } : []
```

Comment:
Could you also add `\n` and `\r\n` as well as indentation strings that mix and match whitespace characters, e.g. "\t\f \r\n" might be a valid indentation block (unless we decide we want to disallow newline characters)

Reply:
Extra values added.